projects/packages/forms/src/blocks/contact-form/editor.scss

```diff
@@ -491,13 +491,7 @@
 }
 
 .jetpack-field-label__width {
-	.components-button-group {
-		display: block;
-	}
-
-	.components-base-control__field {
-		margin-bottom: 12px;
-	}
+	margin-bottom: 1em;
```

Comment:
I don't think this will be necessary once https://github.com/Automattic/jetpack/pull/41133 is merged.